core/ROB.hpp

```diff
@@ -134,5 +134,12 @@ namespace olympia
         void dumpDebugContent_(std::ostream& output) const override final;
         void onStartingTeardown_() override final;
 
+        void retireSysInst(InstPtr & );
+
+        void setup_scoreboard_view() ;
```

Comment:
Neither of these method names following existing convention nor our coding style guidelines.  Please fix

Reply:
done